packages/salesforcedx-vscode-core/src/commands/forceOrgCreate.ts

```diff
@@ -13,15 +13,22 @@ import * as path from 'path';
 import * as vscode from 'vscode';
 import { nls } from '../messages';
 import {
+  CancelResponse,
+  CompositeParametersGatherer,
+  ContinueResponse,
   FileSelection,
   FileSelector,
+  ParametersGatherer,
   SfdxCommandlet,
   SfdxCommandletExecutor,
   SfdxWorkspaceChecker
 } from './commands';
 
-class ForceOrgCreateExecutor extends SfdxCommandletExecutor<FileSelection> {
-  public build(data: FileSelection): Command {
+export const DEFAULT_ALIAS = 'scratchOrg';
```

Comment:
@ruthemmanuelle - Is this an acceptable default name for scratchOrg? This is going to show up when you do `force:org:list`. Since we are setting this as the default for you if you don't provide one, is this OK or should we go with something more explicit like `defaultScratchOrg`?

Reply:
Thanks for catching that Nick, meant to Ruth ask about this. Josh wanted to stay away from using "default" (and the like?) because if would be confusing to see an org with that alias and then see another org such as myFoo set as `defaultusername`. Not sure if it applies to defaultScratchOrg but something to think about

Reply:
I'd like to set an alias that's not likely to overwrite an alias the user has already specified. Would something like `vscodeScratchOrg` work?

Reply:
That works for me